### PR TITLE
Arq 1237 port prober

### PR DESCRIPTION
**Pull request by CSchulz**
Implementation of https://issues.jboss.org/browse/ARQ-1237

### CODE REVIEW

**Comment by aslakknutsen** [comment]
Hmm, what kinda dependency tree does selenium-remote-driver pull in? Warp doesn't technically need Selenium to run, e.g. you can Warp pure REST services etc.. 

Maybe 'clone' PortProber into Warp instead of the dependency?

**Comment by CSchulz** [comment]
@aslakknutsen The CI having the same issue like I have local. Is there anything broken with the build itself?
If I revert all my changes it breaks too.

**Comment by aslakknutsen** [comment]
@CSchulz Think you should adjust the license header check instead of adding the 'other' license header.. :) Exclude the files from the check

**Comment by CSchulz** [comment]
@aslakknutsen Anything else? :smiley:

**Comment by aslakknutsen** [comment]
Think that should do. :) @lfryc ?

File: impl/src/main/java/org/jboss/arquillian/warp/impl/client/proxy/ProxyURLProvider.java [hunk 1]
```diff
@@ -54,6 +55,9 @@
     @Inject
     private Instance<TestClass> testClass;
 
+    @Inject
+    private Instance<URLMapping> urlMapping;
+
```
Comment (lfryc) [inline impl/src/main/java/org/jboss/arquillian/warp/impl/client/proxy/ProxyURLProvider.java]:
````
private URLMapping urlMapping() {
         return serviceLoader.get().onlyOne(URLMapping.class);
     }
````

is no longer needed

**Comment by lfryc** [comment]
@aslakknutsen I wonder if we should add a note about classes that were adopted from Selenium project, wdyt?

**Comment by lfryc** [comment]
Otherwise looks like nice addition, thanks @CSchulz !

**Comment by aslakknutsen** [comment]
@lfryc Sure, we can add a NOTICE file with a ref to the classes copyrighted to SFC from the Selenium project. All same license so not technically needed, but a reference is always nice :)

**Comment by lfryc** [comment]
@aslakknutsen yep, just for reference; I will add it and merge this!

**Comment by lfryc** [comment]
Merged, thanks @CSchulz !

https://github.com/arquillian/arquillian-extension-warp/commit/006f6a50ae153c7deac3b00a5b06e1c6f709d412

**Pull request closed without merging**